x-pack/plugins/alerting/public/plugin.ts

```diff
@@ -57,6 +57,7 @@ export interface PluginSetupContract {
 }
 export interface PluginStartContract {
   getNavigation: (ruleId: Rule['id']) => Promise<string | undefined>;
+  getMaxAlertsPerRun: () => number | undefined;
```

Comment:
is this function ever called? not seeing any functional references to it ...

Reply:
We call it [here](https://github.com/elastic/kibana/pull/179680/files#diff-a3359a0b989ca2039b8e25ab412c67bf40143499be084515d8248125c338b741R36) in security solution, it basically acts as a getter for the config value and this defines its type.

Reply:
thx - VSCode didn't show this in find results, till I opened that file (yikes!)